blog/Webots-2018-a-release.md

```diff
@@ -1,10 +1,12 @@
 # Version R2018a Released
 
-<p id="publish-data">By XXX - XXth XXX 2018</p>
+<p id="publish-data">By Tom Norton - XXth XXX 2018</p>
 
 ---
 
-Today we're happy to announce the release of Webots R2018a. This new release brings a ton of new features and improvements, coupled with the same dedicated patches addressing bugs and regressions. Listed below are some of the key features of this release (for a full list of changes please refer to the ChangeLog, found [here](https://www.cyberbotics.com/dvd/common/doc/webots/ChangeLog.html)).
+Today we're happy to announce the release of the all-new Webots 2018a, packed with some great new features. We're going to talk about some of them here, but for a comprehensive list of changes please refer to the ChangeLog, found [here](https://www.cyberbotics.com/dvd/common/doc/webots/ChangeLog.html).
+
+This release brings a change to the way Webots is released. We have been looking for a means of improving our release schedule to provide reliability for our users, and a more strict release schedule that ensures updates are shipped predictably and reliably. Thus, we are migrating from a Major-Minor-Maintenance versioning system to follow the MathWorks model. We will now be aiming to release new editions of Webots annualy, with major intermediate releases every six months, starting with Webots 2018a, then Webots 2018b, then Webots 2019a etc. Maintenance revisions will still be shipped to fix critical bugs between these release milestones.
```

Comment:
2018a &rarr; R2018a (same for R2018b and R2019a). Also I would not mention MathWorks, but rather explain it. For example: "Thus, we are migrating to from Major-Minor-Maintenance to a Year-Release-Mainteance versioning system."